modules/rules/variables.tf

```diff
@@ -0,0 +1,93 @@
+variable "scc_instance_id" {
+  type        = string
+  description = "ID of the SCC instance in which to create the attachment."
+  default     = "5cd2ee42-a26b-4631-a0d6-e0f220716d5f"
```

Comment:
Description is wrong (not attachment) and I am not sure if we should have a default for this?

Reply:
this should not have any default value